### ISSUE TITLE
Multiple pictures to drop and share as a simple Gallery 

### ISSUE THREAD

**Issue by mrg358** [rubikscraft/Picsur#16]
**Feature request**
Only one feature I really miss is "multiple pictures to drop and share as a simple gallery".  Many file sharing self-hosted options out there, but none of them has what I am looking for. This would be so, so nice to have :) 
Thanks for the good job so far.

**Comment by rubikscraft** [comment]
This has already been suggested in another issue, but I'll leave it up since the main issue there is something else. 
But I'm working on it right now, its taking quite some work tho to make sure that you don't run out of resources when trying to upload a shitton of images.

**Comment by mrg358** [comment]
Just a suggestion, not sure if helpful. Maybe an option to limit the number of uploaded files and do the upload in batches. User configured option. E.g. a low spec Picsur instance could set up upload limited to 10 or even just 5 files at once. Important thing would be to keep the option to add files into existing albums later on, until user decides to close the album.

I know this doesn't entirely solve the problem of overload or abuse the system, but I think some kind of user defined limits would be reasonable to have.

**Comment by rubikscraft** [comment]
That is a thing I could do, but the problem there is that it only works if you know only one user is using the system. Since it supports multiple users you could still have a situation where multiple users upload the full maximum of images, and manage to go over the system resources.
I'm currently trying to implement a queueing system that schedules conversions as long as there are resources available. That should hopefully be more reliable.

**Comment by mrg358** [comment]
You are right, I was thinking about these limits in a single user instance. Basically what I really meant was admin defined limits, not user defined. Only the admin should decide how much load his instance could take.
Queueing system sound just perfect, I am really looking forward to this one.
